.github/workflows/scriptcheck.yml

```diff
@@ -154,9 +154,22 @@ jobs:
         env:
           PYTHONPATH: ./addons
 
-      - name: test htmlreport
+      - name: test htmlreport (standalone)
         run: |
-          htmlreport/test_htmlreport.py
+          test/tools/htmlreport/test_htmlreport.py
+          cd htmlreport
+          ./check.sh
```

Comment:
This still needs to be moved to `test/tools/htmlreport`

Reply:
Should be updated.